starport/services/chain/chain.go

```diff
@@ -182,9 +182,25 @@ func New(path string, options ...Option) (*Chain, error) {
 		)
 	}
 
-	// append keyring backend if specified
-	if c.options.keyringBackend != "" {
+	// use keyring backend if specified
+	if c.options.keyringBackend != chaincmd.KeyringBackendUnspecified {
 		ccoptions = append(ccoptions, chaincmd.WithKeyringBackend(c.options.keyringBackend))
+	} else {
+		// check if keyring backend is specified in config
+		config, err := c.Config()
+		if err != nil {
+			return nil, err
+		}
+		if config.Init.KeyringBackend != "" {
+			configKeyringBackend, err := chaincmd.KeyringBackendFromString(config.Init.KeyringBackend)
+			if err != nil {
+				return nil, err
+			}
+			ccoptions = append(ccoptions, chaincmd.WithKeyringBackend(configKeyringBackend))
+		} else {
+			// default keyring backend used is OS
+			ccoptions = append(ccoptions, chaincmd.WithKeyringBackend(chaincmd.KeyringBackendOS))
```

Comment:
Can we set this here https://github.com/tendermint/starport/pull/637/files#diff-38e980d06703d9813d916e0e7da1b48e50c48e4327f13b7e950eadcedefb1fbbL128?